.chloggen/fix-ca-cert-renewal-race.yaml

```diff
@@ -0,0 +1,19 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: target allocator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix CA certificate renewal race condition by extending CA certificate duration to 1 year
```

Comment:
this changelog message seems out of step with the changes. The CA certificate duration is set to 720 days, roughly 2 years. Am I missing something?